Makefile

```diff
@@ -123,7 +123,9 @@ install-core: all
 	$(INSTALL) -m 0644 etc/stbt.conf \
 	    $(DESTDIR)$(sysconfdir)/stbt/stbt.conf
 	for filename in $(INSTALL_CORE_SCRIPTS); do \
-	    $(INSTALL) -m 0755 $$filename $(DESTDIR)$(libexecdir)/stbt/$$filename; \
+	    sed '1s,^#!/usr/bin/python\b,#!/usr/bin/python$(python_version),' \
+	        $$filename > $(DESTDIR)$(libexecdir)/stbt/$$filename; \
+	    chmod 0755 $(DESTDIR)$(libexecdir)/stbt/$$filename; \
```

Comment:
An alternative would be to modify the stbt shell script that we already modify on installation.

Reply:
I tried, but we also have to teach `stbt-screenshot` which python to use when it calls `stbt_run.py`.... which isn't unfixable but I've run out of steam.